### PR TITLE
Update Eset.xml

### PR DESCRIPTION
**Pull request by volatile-Daniel**
We need it for proper work of our security tool

### CODE REVIEW

**Comment by volatile-Daniel** [comment]
Mentioned exclusion pattern address is processed internally and AFAIK there is no public url, which is able to cover this test.

**Comment by jeremyn** [comment]
https://eset.com/BPPRedirector looks valid to me. What is the specific problem you're trying to fix? Maybe we can think of another way to handle it.

**Comment by Hainish** [comment]
@volatile-Daniel can you change the comment to be "page redirected on server side, and this breaks ESET Secure Browser"

Other than that, looks good to me

File: src/chrome/content/rules/Eset.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
 
 	<!--	Direct rewrites:
 				-->
-	<target host="eset.com" />
+	<!--<target host="eset.com" /> page redirected on server side, and this breaks ESET Secuce Browser -->
```
Comment (Hainish) [inline src/chrome/content/rules/Eset.xml]: "Secure" not "Secuce", right?

**Pull request merged**